docs/content/how-to/aws/index.md

```diff
@@ -0,0 +1,555 @@
+---
+title: AWS Architecture and Resources
+---
+
+# AWS Architecture and Resources
+
+In this section we will talk about the AWS Resources and Openshift objects generated by Hypershift and how the Controllers interact with them during a Hosted Cluster deployment.
+
+## Prerequisites and Assumptions
+
+Here we will set the legend about the terms used in this document:
+
+- **The Management Cluster**: It's the cluster which will host the Hosted Control Planes and the operator itself.
+- **Guest Clusters**: Those are the Hosted Control Plane deployed inside of the Management cluster.
+- **Node Pools**: Set of nodes (0..N) which belongs to one Hosted Cluster. Once we have at least 1 worker inside of this NodePool we will have at least 1 EC2 Instance mapped in AWS. but the relationship goes in reverse, the NodePool does not knows about the Nodes or Workers. On the other hand we have the *Machines* which has a Label where the *NodePool* and *MachineSet* is configured.
+
+In the Prerequisites side we will have 2 states:
+
+- Prerequisites to deploy and work with Hypershift Operator
+- Prerequisites to deploy a HostedCluster
+
+### Prerequisites to deploy and work with Hypershift Operator
+
+- **OCP Cluster Admin account**: Regarding roles, this is required in order to allow the CLI to generate the proper SA and ClusterRoles.
+- **AWS Route 53 Domain**: This is needed in order to create reachable API via DNS name instead of using the ELB name.
+- **S3 Bucket**: Publicly accesible, this is mandatory in order to upload the OIDC data for the given HostedCluster.
+- **OCP Pull Secret**: This is not an AWS Object but will be mandatory to deploy the OCP CLuster
+- **AWS Credentials**: To create resources into AWS Provider
```

Comment:
What kind of credentials (ignore if you describe it later, I am reviewing linearly :)